src/events.js

```diff
@@ -37,29 +65,16 @@ module.exports = function(Chart) {
 		// Detect hover events
 		if (e.type === 'mousemove') {
 			if (element && !element.hovering) {
+				// end hover on the last hovered element
+				if(lastHoveredElement && element !== lastHoveredElement) {
```

Comment:
the formatting here doesn't quite match what's expected. please run `gulp lint` after rebasing to check whether there are any style issues